Packages/com.unity.inputsystem/InputSystem/SourceGenerators/Source~/SourceGenerators/InputActionsSyntaxReceiver.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Unity.InputSystem.SourceGenerators;
+
+public class InputActionsSyntaxReceiver : ISyntaxReceiver
```

Comment:
What does this do - can we add a small comment to explain this class

Reply:
This is removed now, but it is a Roslyn interface for visiting syntax nodes.